### PR TITLE
Feature/add forklift gui

### PR DESCRIPTION
**Pull request by ShaneIsrael**
It does stuff and things

### CODE REVIEW

File: plugins/gui/bin/www [hunk 1]
```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app');
+var debug = require('debug')('Noise:server');
```
Comment (dcshock) [inline plugins/gui/bin/www]: Noise shouldn't be referenced.

File: plugins/gui/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM build.sofi.com:5000/sofi-node-base
```
Comment (dcshock) [inline plugins/gui/Dockerfile]: This can't be built outside of the SoFi docker repo. Can you build a public node base container?

**Comment by dcshock** [comment]
For the most part the interface is headed where it should be. Let's correct the SoFi and noise dependencies before merging. Once those are cleaned up I'll get this merged down with some minor cleanups.

**Comment by ShaneIsrael** [comment]
Alright sounds good.

**Pull request merged**